frontend/app/common/fetcher.ts

```diff
@@ -46,6 +46,9 @@ const createFetcher = (baseUrl: string = ''): Methods => {
       headers[JWT_HEADER] = activeJwtToken;
     }
     headers[XSRF_HEADER] = getCookie(XSRF_COOKIE) || '';
+    // An HTTP header cannot be empty.
+    // Although some webservers allow this (nginx, Apache), others answer 400 Bad Request (lighttpd).
+    if (headers[XSRF_HEADER] === '') delete headers[XSRF_HEADER];
```

Comment:
@forkbomb9 why not store `getCookie(XSRF_COOKIE)` into a temporary value and test that it exists and is not empty before setting `headers[XSRF_HEADER]`.  That way, you can skip the `delete headers[XSRF_HEADER]`

Reply:
Yep, will do that. I'll also fix the test suite